README.md

```diff
@@ -15,34 +18,144 @@ pitfalls.
+|  `--language_in VAL` |  Sets the language spec to which input sources should conform. Options: `ECMASCRIPT3`, `ECMASCRIPT5`, `ECMASCRIPT5_STRICT`, `ECMASCRIPT6_TYPED` (experimental), `ECMASCRIPT_2015`, `ECMASCRIPT_2016`, `ECMASCRIPT_2017`, `ECMASCRIPT_2018`, `ECMASCRIPT_2019`, `STABLE`, `ECMASCRIPT_NEXT`  |
+|  `--language_out VAL`  |  Sets the language spec to which output should conform. Options: `ECMASCRIPT3`, `ECMASCRIPT5`, `ECMASCRIPT5_STRICT`,`ECMASCRIPT_2015`, `ECMASCRIPT_2016`, `ECMASCRIPT_2017`, `ECMASCRIPT_2018`,`ECMASCRIPT_2019`, `STABLE` |
+| <code>--warning_level (-W) [QUIET &#124; DEFAULT &#124; VERBOSE]</code>  | Specifies the warning level to use.  |
+
+#### See the [Google Developers Site](https://developers.google.com/closure/compiler/docs/gettingstarted_app) for documentation including instructions for running the compiler from the command line.
+
+### NodeJS API
+You can access the compiler in a JS program by importing
+`google-closure-compiler`:
+
+```javascript
+import closureCompiler from 'google-closure-compiler';
+const { compiler } = closureCompiler;
+
+new compiler({
+  js: 'file-one.js',
+  compilation_level: 'ADVANCED'
+});
+```
+
+This package will provide programmatic access to the native Graal binary in most
+cases, and will fall back to the Java version otherwise.
+
+#### Please see the [closure-compiler-npm](https://github.com/google/closure-compiler-npm/tree/master/packages/google-closure-compiler) repository for documentation on accessing the compiler in JS.
+
+## Compiling Multiple Scripts
+
+If you have multiple scripts, you should compile them all together with one
+compile command.
+
+```bash
+google-closure-compiler in1.js in2.js in3.js --js_output_file out.js
+```
+
+You can also use minimatch-style globs.
+
+```bash
+# Recursively include all js files in subdirs
+google-closure-compiler 'src/**.js' --js_output_file out.js
+
+# Recursively include all js files in subdirs, excluding test files.
+# Use single-quotes, so that bash doesn't try to expand the '!'
+google-closure-compiler 'src/**.js' '!**_test.js' --js_output_file out.js
+```
+
+The Closure Compiler will concatenate the files in the order they're passed at
+the command line.
+
+If you're using globs or many files, you may start to run into problems with
+managing dependencies between scripts. In this case, you should use the [Closure
+Library](https://developers.google.com/closure/library/). It contains functions
+for enforcing dependencies between scripts, and Closure Compiler will re-order
+the inputs automatically.
+
+## Getting Help
+
+1.  Post in the [Closure Compiler Discuss
+    Group](https://groups.google.com/forum/#!forum/closure-compiler-discuss).
+1.  Ask a question on [Stack
+    Overflow](https://stackoverflow.com/questions/tagged/google-closure-compiler).
 1.  Consult the [FAQ](https://github.com/google/closure-compiler/wiki/FAQ).
 
-## Building it Yourself
+## Building
```

Comment:
Because this is a compiler, we should be clear that we're building the compiler itself, not building JS with the compiler.